fuzz-tests/fuzz.sh

```diff
@@ -107,7 +107,7 @@ function fuzzer_libfuzzer() {
     set -x
     cargo +nightly fuzz $cmd \
         --release \
-        --no-default-features --features std,libfuzzer-sys \
+        --no-default-features --features std,libfuzzer-sys,post_run_db_check\
```

Comment:
no need to add `post_run_db_check` when building AFL (line 135)?

Reply:
ah good point